toonz/sources/image/ffmpeg/tiio_ffmpeg.cpp

```diff
@@ -9,6 +9,7 @@
 #include <QRegExp>
 #include "toonz/preferences.h"
 #include "toonz/toonzfolders.h"
+#include "toonzqt/dvdialog.h"
```

Comment:
I still did not reproduce the issue but the Travis CI claimed as follows:
>In file included from >/Users/travis/build/opentoonz/opentoonz/toonz/sources/image/ffmpeg/tiio_ffmpeg.cpp:12:
>/Users/travis/build/opentoonz/opentoonz/toonz/sources/include/toonzqt/dvdialog.h:11:10: fatal error: 'QDialog' file not found
>#include \<QDialog\>

Replacing this line to `#include "tmsgcore.h"` seems to fix the error.